docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -220,7 +221,7 @@ can use [kubeadm config](kubeadm-config.md).
 
 ## kubeadm alpha phase addon {#cmd-phase-addon}
 
-You can install all the available addons with the `all` sub command, or 
+You can install all the available addons with the `all` subcommand, or 
 install them selectively.
 
```

Comment:
Add something related to CoreOS?
Add something related to SupportIPVSProxyMode?

Reply:
What would be CoreOS-specific?
We don't support that feature gate anymore